docs/sources/dashboards/build-dashboards/create-dashboard-url-variables/index.md

```diff
@@ -20,7 +20,7 @@ weight: 250
 # Dashboard URL variables
 
 Grafana can apply variable values passed as query parameters in dashboard URLs.
-For more information, refer to [Manage dashboard links]({{< relref "../manage-dashboard-links/" >}}) and [Templates and variables]({{< relref "../../variables/" >}}).
+For more information, refer to [Manage dashboard links][] and [Templates and variables][].
```

Comment:
[Templates and variables] reference is not rendering as a link in the build, but just as the raw text. 
```suggestion
For more information, refer to [Manage dashboard links][] and [Templates and variables][].
```

Reply:
Good catch, fixed in 62139350fc